worker/httpserver/worker.go

```diff
@@ -169,9 +169,11 @@ func (w *Worker) loop() error {
 		}
 	}()
 	defer func() {
+		ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
```

Comment:
let's define the timeout as a const in the worker manifold and set as per other worker config
is 30s enough? sometimes we need a little longer i think

Reply:
Decided to go with a var so that it can be changed in tests